allure_combine/combine.py

```diff
@@ -267,14 +267,21 @@ def combine_allure(folder, dest_folder=None, remove_temp_files=False, auto_creat
                 tag.replaceWith(full_script_tag)
 
     print("Done")
+    
+    if not file_name:
```

Comment:
You dont need to copy and paste all saving procedure in your "if".
You can write just:
```
if file_name is None:
    file_name = dest_folder + f"{sep}complete.html"
```
and use files_name instead of complete.html farther.